tests/util/generateBatchPlateData.ts

```diff
@@ -0,0 +1,114 @@
+import * as fs from 'fs'
+import completeHGVTechRecord from '../resources/techRecordCompleteHGVPlate.json';
+import completeTRLTechRecord from '../resources/techRecordsTrlPost.json';
+import existingSeed from '../resources/technical-records-v3.json';
+import { TechRecordGETHGV, TechRecordGETTRL } from '@dvsa/cvs-type-definitions/types/v3/tech-record/tech-record-verb-vehicle-type';
+
+function padZeroes(number: string, size: number) {
+    number = number.toString();
+    while (number.length < size) number = "0" + number;
+    return number;
+}
+
+let records = []
+let triggerData = []
+console.log("starting")
```

Comment:
a lot of the functions dont use console logging but use the winston logger instead

Reply:
This is really a throwaway script to show how the test data was generated. These logs are purely for tracking progress when invoking the file through `ts-node` in the CLI.